doc/asciidoc/kafka-connect/index.adoc

```diff
@@ -10,6 +10,54 @@ The Neo4j Streams project provides a Kafka Connect plugin that can be installed
 - Ingest data from Kafka topics directly into Neo4j via templated Cypher queries;
 - Stream Neo4j transaction events (*coming soon*).
 
+=== Plugin installation
+
+You can choose your preferred way in order to install the plugin
+
+==== Build it locally
+
+Build the project by running the following command:
+
+    mvn clean install
+
+Inside the directory `<neo4j-streams>/kafka-connect-neo4j/target/component/packages` you'll find a file named `neo4j-kafka-connect-neo4j-<VERSION>.zip`, please place it into your Kafka Connect `plugins` dir.
+
+==== Download the zip from the Confluent Hub
```

Comment:
can we flip this around confluent hub first then build locally?
for build locally we should also add the git clone, cd to right directory and then mvn